src/org/thoughtcrime/securesms/ConversationUpdateItem.java

```diff
@@ -55,9 +55,8 @@ public void bind(@NonNull DcMsg                   messageRecord,
                    @NonNull Recipient               conversationRecipient,
                             boolean                 pulseUpdate)
   {
-    bind(messageRecord, dcChat, batchSelected);
+    bind(messageRecord, dcChat, batchSelected, pulseUpdate);
     setGenericInfoRecord(messageRecord);
-    setSelected(batchSelected.contains(messageRecord));
```

Comment:
What does this change do?

Reply:
that is what adds support for the "pulse" effect in both normal messages and system messages, previously it was implemented only for normal messages, I moved that to the parent base class

Reply:
Do you know why the `setSelected(batchSelected.contains(messageRecord));` was there before?

Reply:
@Hocuri yes, it was to mark message as selected when you are in "selection mode" (to put the light blue background) that logic is inside `BaseConversationItem.setInteractionState()` which takes care of the interaction state (pulse effect and selected row)